3-extra/1card-validator.js

```diff
@@ -0,0 +1,44 @@
+//To tray the functions write   npm run extra-card   in the terminal
+
+// onlyNumber is for check if in the string are only numbers
+const onlyNumber = (number) => /^[0-9]{16}$/.test(number);
+
+// oddLastNumber check if the last number is odd
+const oddLastNumber = (number) => /[02468]$/.test(number);
+
+// notOnlyOneNumber add all the number in a Set and if is more than one mean is not only one value
+const notOnlyOneNumber = (number) => new Set([...String(number)]).size !== 1;
+
+// moreThanSixteen check if sum of all the number is more than 16
+const moreThanSixteen = (number) =>
+  [...String(number)].reduce((a, b) => +a + +b) > 16;
```

Comment:
`+a + +b` looks awful! :) Why don't you use a simpler version: `a + b`?

Reply:
I did like this because are strings, to convert the string to number.  
 Number(a)+Number(b)
maybe this is better?